src/authors/scss/style.scss

```diff
@@ -0,0 +1,4 @@
+@import '_variables.scss';
+@import '_body.scss';
+@import '_footer.scss';
+@import '_media.scss'
```

Comment:
Didn't we discuss aggregating it into one file `src/authors/authors.scss`? Why you didn't do it?